inc/helpers.php

```diff
@@ -0,0 +1,123 @@
+<?php
+/**
+ * Helper functions for Ads.txt.
+ *
+ * @package Ads_Txt_Manager
+ */
+
+namespace AdsTxt;
+
+/**
+ * Display the contents of /ads.txt when requested.
+ *
+ * @return void
+ */
+function tenup_display_ads_txt() {
```

Comment:
Now that these functions have been moved to a file with a namespace, I'd suggest we remove any namespacing from the function names. This includes any of the `tenup_` prefixes and could also remove some of the `adstxt` ones.

I'd suggest something like the following:

- `tenup_display_ads_txt` -> `display_ads_txt`
- `add_adstxt_capabilities` -> `add_capabilities`
- `remove_adstxt_capabilities` -> `remove_capabilities`
- `tenup_ads_txt_add_query_vars` -> `add_query_vars`

Reply:
Thanks @dkotter, I've made the suggested updates here: https://github.com/10up/ads-txt/pull/149/commits/df602d222f5edebe331b12b69e52f0354fb35126